src/redux/reducers/TodoReducer.ts

```diff
@@ -3,18 +3,19 @@ import { AddItemPayload } from '../actions/todoActions/addItemAction';
 import { ItemDeletePayload } from '../actions/todoActions/deleteItemAction';
 import { TodoActionTypes } from '../actions/todoActions';
 import { DoneItemPayload } from '../actions/todoActions/doneItemAction';
-import { SignInPayload } from '../actions/todoActions/signInAction';
+import { AuthActionsTypes } from '../actions/authActions';
+import { SignInPayload } from '../actions/authActions/signInAction';
 
 export type TodoReducerState = {
   todoItems: TodoItemType[];
   userToken: SignInPayload['userToken'] | null;
-  user: SignInPayload['user'] | null;
+  user: SignInPayload['user'];
 };
 
 const initialState: TodoReducerState = {
   todoItems: [],
   userToken: null,
-  user: null,
+  user: undefined,
```

Comment:
u can just remove this line, result will be the same

Reply:
in type AuthReduserState i have user that's why i need this line

Reply:
1) you shouldn't close conversation if you haven't fix it and need some additional discussion 
2) you should change your type `TodoReducerState` and then you can remove this 
another question
why do you need user in `TodoReducerState`? I think you store user in `AuthReducerState`